doc/img/overview.puml

```diff
@@ -21,7 +21,7 @@ Lambda(app1, "Aggregator", "...")
 Lambda(app2, "Component", "...")
 Lambda(app3, "...", "...")
 Lambda(appApiTest, "api-test", "hello from lambda")
-Lambda(authorizer, "authorizer", "bearer token validation")
+Lambda(authorizer, "authorizer", "api key validation")
```

Comment:
Put authorizer below API gateway, as it is more a component of the API gateway rather than a function providing part of the functionality behind the API?